build.gradle

```diff
@@ -1,23 +1,24 @@
 buildscript {
-    repositories {
+	repositories {
 		mavenLocal()
-        mavenCentral()
-		jcenter()	
+		mavenCentral()
+		jcenter()
 		maven {
-			name = "gt"
-			url = "https://gregtech.overminddl1.com/"
-		}	
-        maven {
-            name = "sonatype"
-            url = "https://oss.sonatype.org/content/repositories/snapshots/"
-        }
+			name = "jitpack"
+			url = "https://jitpack.io"
```

Comment:
What do you need Jitpack for? Forge is pulled from maven.minecraftforge.net (which you defined below)

Reply:
I don't actually know, I was just copying things in until it built - but looks like that wasn't needed.

Reply:
hmm how do I indicate that 1142e97 below resolves this?